scripts/gitlab/build.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
```

Comment:
Are we even using this `build.sh` monstrosity?
I think at least partially the idea was to replace it with smaller and more straightforward task-specific scripts (which it seems like you did). Also, `build.sh` is never mentioned in the `gitlab-ci.yml`